lib/ruby_warrior/decree.rb

```diff
@@ -0,0 +1,10 @@
+require 'active_record'
+
+module RubyWarrior
+  class Decree < ActiveRecord::Base
```

Comment:
Pulled the name `Decree` out of nowhere because the King is involved, but probably not the best fit. Thoughts?

Reply:
What about MapObject?

Since this is what the contents of the table will describe